tests/publiccloud/check_registercloudguest.pm

```diff
@@ -148,6 +149,49 @@ sub check_instance_unregistered {
     }
 }
 
+sub new_registration {
+    my ($instance) = @_;
+    record_info('Starting registration...');
+    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
```

Comment:
see my [comment above](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20260/files#r1780624961)

Reply:
it is expected behavior , this way we checking that in PAYG call of registercloudguest not doing any mess and recognize that system is correctly registered . So it is some kind of verification of validity of PAYG registration